src/schema/v2/edition_set.ts

```diff
@@ -43,10 +47,31 @@ export const EditionSetType = new GraphQLObjectType<any, ResolverContext>({
       type: GraphQLString,
       resolve: ({ editions }) => editions,
     },
+    heightCm: {
+      description:
+        "If you need to render artwork dimensions as a string, prefer the `Artwork#dimensions` field",
+      type: GraphQLFloat,
+      resolve: ({ height_cm }) => height_cm,
+    },
+    internalDisplayPrice: {
+      type: GraphQLString,
+      resolve: ({ internal_display_price }) => internal_display_price,
+      description:
+        "Price for internal partner display, requires partner access",
+    },
     isAcquireable: {
       type: GraphQLBoolean,
       resolve: ({ acquireable }) => acquireable,
     },
+    isForSale: {
+      type: GraphQLBoolean,
+      resolve: ({ forsale }) => forsale,
+    },
+    isInquireable: {
+      type: GraphQLBoolean,
+      description: "Do we want to encourage inquiries on this work?",
+      resolve: ({ inquireable }) => inquireable,
```

Comment:
I don't see this in the edition set JSON - https://github.com/artsy/gravity/blob/b7ea76c854c1d79175f89daa3e4e810a2b722d8c/app/models/domain/edition_set.rb#L82

Reply:
Resolved to the artwork item and noted in description